docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,58 @@
+Status
+~~~~~~
+Draft
```

Comment:
Consider starting with "Accepted", unless it needs to change in order to merge. :)